inst/include/interface/rcpp/rcpp_objects/rcpp_tmb_distribution.hpp

```diff
@@ -288,6 +347,66 @@ class DlnormDistributionsInterface : public DistributionsInterfaceBase {
     return dlnorm.evaluate();
   }
 
+    virtual void finalize() {
+        if (this->finalized) {
+            //log warning that finalize has been called more than once.
+            FIMS_WARNING_LOG("LogNormalLPDF  " + fims::to_string(this->id_m) + " has been finalized already.");
+        }
+
+        this->finalized = true; //indicate this has been called already
+
+
+        std::shared_ptr<fims_info::Information<double> > info =
+                fims_info::Information<double>::GetInstance();
+
+        fims_info::Information<double>::density_components_iterator it;
+
+        //search for density component in Information
+        it = info->density_components.find(this->id_m);
+        //if not found, just return
+        if (it == info->density_components.end()) {
+            FIMS_WARNING_LOG("LogNormalLPDF " + fims::to_string(this->id_m) + " not found in Information.");
+            return;
+        } else {
+            std::shared_ptr<fims_distributions::LogNormalLPDF<double> > dnorm =
```

Comment:
chage dnorm to dlnorm her and on lines 374 and 376

Reply:
This is now fixed in 932beb97